snapshot/snapshotfs/upload.go

```diff
@@ -538,6 +538,9 @@ func (u *Uploader) processChildren(
 ) error {
 	var wg workshare.AsyncGroup
 
+	// ensure we wait for all work items before returning
+	defer wg.Close()
```

Comment:
does `snapshot/snapshotfs/snapshot_tree_walker.go` also need to call `Close()`?

Reply:
done